tests/data/test_history.py

```diff
@@ -133,7 +133,7 @@ def test_load_data_with_new_pair_1min(ohlcv_history_list, mocker, caplog,
     load_pair_history(datadir=tmpdir1, timeframe='1m', pair='MEME/BTC')
     assert file.is_file()
     assert log_has_re(
-        'Download history data for pair: "MEME/BTC", timeframe: 1m '
+        'Download history data for pair: "MEME/BTC" (1/1), timeframe: 1m '
```

Comment:
If you look at CI - you'll notice the follwoing line:
```
INFO     freqtrade.data.history.history_utils:history_utils.py:182 Download history data for pair: "MEME/BTC" (0/1), timeframe: 1m and store in C:\Users\runneradmin\AppData\Local\Temp\pytest-of-runneradmin\pytest-0\test_load_data_with_new_pair_10.
```

it's clearly showing the mistake (it's not a problem, either).
you're printing the log before the download - so the first (and only) pair will be `0/1` - not `1/1`.

Reply:
thank you why I cant see it befour... 🥲

Reply:
Yes, it's called [documentation](https://www.freqtrade.io/en/stable/installation/)